pest/src/position.rs

```diff
@@ -16,8 +16,7 @@ use std::str;
 
 use span;
 
-/// A `struct` containing a position that is tied to a `&str` which provides useful methods to
-/// manually parse it.
+/// A character position in a `&str` which provides useful methods to manually parse that string.
```

Comment:
I guess this should explain the fact that the `Position` is actually a pointer that points *between* two characters and not a character itself.